tsconfig.json

```
@@ -19,7 +19,9 @@
       "@mui-internal/api-docs-builder": ["./node_modules/@mui/monorepo/packages/api-docs-builder"],
       "@mui-internal/api-docs-builder/*": [
         "./node_modules/@mui/monorepo/packages/api-docs-builder/*"
-      ]
+      ],
+      "docs-base/*": ["./docs/*"],
+      "docs/*": ["./node_modules/@mui/monorepo/docs/*"]
```

```suggestion
      "docs/*": ["./docs/*"],
      "docs-core/*": ["./node_modules/@mui/monorepo/docs/*"]
```
Why not do this? It's more intuitive for Base UI's developers and it would create less changes (or maybe I am wrong :))

---

This was my thought initially as well. But the shared docs import from `docs` internally, and this would cause them to import from Base UI docs, which is far from what we want.

---

Agree with Michal

---

Makes sense 👍